src/os/ui/menu/listmenu.js

```diff
@@ -127,18 +127,18 @@ os.ui.menu.list.handleListEvent = function(event) {
     var eventType = event.type.replace(os.ui.menu.list.PREFIX_REGEXP, '');
     switch (eventType) {
       case os.action.EventType.SELECT:
-        // omit selection events from the stack to reduce clutter
-        new os.command.SelectAll(source.getId()).execute();
+        // don't create and execute a command if it's simple and we don't want it on the stack
+        if (source) source.selectAll();
```

Comment:
Might be cleaner to wrap this entire thing in `if (source)` if there's a chance that it can be null. All of these calls use it.

Reply:
Removed!  There was a check above it already.